faasmcli/faasmcli/util/policy.py

```diff
@@ -0,0 +1,218 @@
+                            if _char == ">":
+                                if _count == 0:
+                                    _start = _index
+                                _count += 1
+                            if _char == "<":
+                                _count -= 1
+                                if _count == 0:
+                                    _end = _index
+                                    break
+                        cur = _end + 1
+                        new = reversed_execution_stack[_start:_end][::-1][:-1]
+                        _found = False
+                        for _id, _label in enumerate(list(reversed(self._verify_chain[last_writer_func][str(int(last_writer_id))]['nodes'][:int(last)]))):
+                            if not _label.startswith('write'):
+                                last = last -1 - _id
+                                _found = True
+                                break
+                        if not _found:
+                            return False
+                        _split = new.split(':')
+                        if self._verify_chain[last_writer_func][str(int(last_writer_id))]['nodes'][last] != '{}:{}'.format(_split[0], _split[2]):
+                            return False
+                        _split = new.split(':')
+                        new = '<{}>'.format(new)
+                        if not self._is_valid_execution(new, _split[0], _split[2]):
+                            return False
+                        internal = True
+                    elif char == ",":
+                        cur += 1
+                    else:
+                        tmp = cur
+                        tmp_internal = False
+                        for _index, _char in enumerate(reversed_execution_stack[cur:]):
+                            if _char == "<":
+                                cur += _index
+                                break
+                            if _char == ',':
+                                tmp_internal = True
+                                cur += _index
+                                break
+                        new = reversed_execution_stack[tmp-1:cur][::-1][:-1]
+                        if internal:
+                            _found = False
+                            for _id, a in enumerate(list(reversed(self._verify_chain[last_writer_func][str(int(last_writer_id))]['nodes'][:int(last)]))):
+                                if not a.startswith('write'):
+                                    last = last - 1 - _id
+                                    _found = True
+                                    break
+                            if not _found:
+                                return False
+                            _split = new.split(':')
+                            if tmp_internal:
+                                if self._verify_chain[last_writer_func][str(int(last_writer_id))]['nodes'][last]!= '{}:{}'.format(_split[0], _split[2]):
```

Comment:
All the long lines seem to be related to accessing the `self._verify_chain` variable, could you perhaps factor out a couple of convenience methods to avoid them?